workers/ohsome_quality_analyst/utils/definitions.py

```diff
@@ -1,35 +1,72 @@
-"""
-Global Variables and Functions.
-"""
-
+"""Global Variables and Functions."""
 import glob
 import logging
 import logging.config
 import os
 import sys
+from dataclasses import dataclass
+from types import MappingProxyType
 from typing import Dict, List
 
 import yaml
+from xdg import XDG_DATA_HOME
 
 from ohsome_quality_analyst import __version__ as oqt_version
+from ohsome_quality_analyst.utils.exceptions import RasterDatasetUndefinedError
 from ohsome_quality_analyst.utils.helper import flatten_sequence, get_module_dir
 
 # Dataset names and fid fields which are available in the Geodatabase
-DATASETS = {
-    "regions": {"default": "ogc_fid", "other": ("name",)},
-    "gadm": {
-        "default": "uid",  # ISO 3166-1 alpha-3 country code
-        "other": (
-            *tuple(("name_{0}".format(i) for i in range(6))),
-            *tuple(("id_{0}".format(i) for i in range(6))),
-            *tuple(("gid_{0}".format(i) for i in range(6))),
-        ),
-    },
-}
+DATASETS = MappingProxyType(  # Immutable dict
+    {
+        "regions": {"default": "ogc_fid", "other": ("name",)},
+        "gadm": {
+            "default": "uid",  # ISO 3166-1 alpha-3 country code
+            "other": (
+                *tuple(("name_{0}".format(i) for i in range(6))),
+                *tuple(("id_{0}".format(i) for i in range(6))),
+                *tuple(("gid_{0}".format(i) for i in range(6))),
+            ),
+        },
+    }
+)
 # Dataset names and fid fields which are through the API
 DATASETS_API = DATASETS.copy()
 DATASETS_API.pop("gadm")
 
+
+@dataclass(frozen=True)
+class RasterDataset:
+    """Raster datasets available on disk."""
+
+    name: str
+    filename: str
+    crs: str
```

Comment:
here you use a string. And I can see below the examples such as "ESRI:54009" or "EPSG:4326".  Can you provide more information why you don't use just the code. e.g. `4326` or `54009`?

Maybe this is defined by pyproj library?

Reply:
Right, thats because of the pyproj lib.
The pyproj lib often does work with integers as well. But not the the `54009` since this does not map to "EPSG:54009" but to "ESRI:54009".
I will add some documentation to the class.

Reply:
pyrpoj will interprete integer as EPSG code and string as an authority string [i.e. `EPSG:4326’]

Reply:
Done